### DISCUSSION TITLE
Node renaming proposal.

### DISCUSSION THREAD

**Discussion by constructor** [cables-gl/cables_docs#552] [Feature Requests] [answered]
If this is too messy or difficult then ignore, but the following makes sense to me.

Currently these two patches are listed with the names:

1. TriggerNumber : Outputs the last number of the input port which was triggered
2. SetNumberOnTrigger : Outputs a number when triggered

The following naming seem clearer to me:

1. TriggerIndex :  Outputs the index of the input port which was triggered
2. TriggerNumber : Outputs a specified number when triggered

As I said, this is not important, just something that came to me when searching for a patch to use and was tricked/confused by the current names.

**Comment by pandrr** [comment, marked as answer]
we agree, will rename with next release
